systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -221,15 +221,17 @@
       obj = obj.updatePlanShift(t_global, kinsol, qp_input, contact_force_detected, next_support);
       qp_input = obj.applyPlanShift(qp_input);
       
-      if(~isempty(obj.joint_pd_override_data))
-        for j = 1:length(obj.joint_pd_override_data.position_ind)
-          qp_input.joint_pd_override(end+1) = struct('position_ind',obj.joint_pd_override_data.position_ind(j),...
-                                                     'qi_des',qp_input.whole_body_data.q_des(obj.joint_pd_override_data.position_ind(j)),...
-                                                     'qdi_des',0,...
-                                                     'kp',obj.joint_pd_override_data.kp(j),...
-                                                     'kd',obj.joint_pd_override_data.kd(j),...
-                                                     'weight',obj.joint_pd_override_data.weight(j));
+      if(~isempty(obj.untracked_joint_ind))
+        for j = 1:length(obj.untracked_joint_ind)
+          qp_input.joint_pd_override(end+1) = struct('position_ind',obj.untracked_joint_ind(j),...
+                                                     'qi_des',q(obj.untracked_joint_ind(j)),...
+                                                     'qdi_des',qd(obj.untracked_joint_ind(j)),...
+                                                     'kp',0,...
+                                                     'kd',0,...
+                                                     'weight',0);
         end
+        qp_input.whole_body_data.q_des(obj.untracked_joint_ind) = q(obj.untracked_joint_ind);
```

Comment:
it's probably worth putting a comment here about why we're setting q_des to q for those joints